src/Tribe/REST/V1/Endpoints/Archive_Category.php

```diff
@@ -0,0 +1,161 @@
+<?php
+
+class Tribe__Events__REST__V1__Endpoints__Archive_Category
+	extends Tribe__Events__REST__V1__Endpoints__Term_Archive_Base
+	implements Tribe__REST__Endpoints__READ_Endpoint_Interface,
+	Tribe__Documentation__Swagger__Provider_Interface {
+
+	/**
+	 * Returns an array in the format used by Swagger 2.0.
+	 *
+	 * While the structure must conform to that used by v2.0 of Swagger the structure can be that of a full document
+	 * or that of a document part.
+	 * The intelligence lies in the "gatherer" of informations rather than in the single "providers" implementing this
+	 * interface.
+	 *
+	 * @link http://swagger.io/
+	 *
+	 * @return array An array description of a Swagger supported component.
+	 */
+	public function get_documentation() {
+		return array(
+			'get' => array(
+				'parameters' => $this->swaggerize_args( $this->READ_args(), array( 'in' => 'query', 'default' => '' ) ),
+				'responses'  => array(
+					'200' => array(
+						'description' => __( 'Returns all the event categories matching the search criteria', 'the-event-calendar' ),
+						'schema'      => array(
+							'title' => $this->get_data_key(),
+							'type'  => 'array',
+							'items' => array( '$ref' => '#/definitions/Term' ),
+						),
+					),
+					'400' => array(
+						'description' => __( 'One or more of the specified query variables has a bad format', 'the-events-calendar' ),
+					),
+					'404' => array(
+						'description' => __( 'No event categories match the query or the requested page was not found.', 'the-events-calendar' ),
+					),
+				),
+			),
+		);
+	}
+
+	/**
+	 * Returns the taxonomy of the terms handled by the endpoint.
+	 *
+	 * @return string
+	 *
+	 * @since TBD
+	 */
+	public function get_taxonomy() {
+		return Tribe__Events__Main::TAXONOMY;
+	}
+
+	/**
+	 * Returns the archive base REST URL
+	 *
```

Comment:
Add `@since TBD` to all of the methods in here. ` Also, `@since` should follow the summary/description of the method according to the [WP DocBlock Formatting standards](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#1-functions-class-methods).